android/app/src/main/java/io/coursepick/coursepick/view/MainActivity.kt

```diff
@@ -130,6 +138,13 @@ class MainActivity :
             ).show()
     }
 
+    override fun copyClientId() {
+        val clipboard = getSystemService(Context.CLIPBOARD_SERVICE) as ClipboardManager
```

Comment:
앞에 `Context`를 붙여둔 이유가 있을까요? 회색 처리 되어있어서요. 👀 

```suggestion
        val clipboard = getSystemService(CLIPBOARD_SERVICE) as ClipboardManager
```

Reply:
저한테는 흰색으로 뜨던데, 없어도 되는 부분이었군요 👀 제거했습니다! cd8e136bcf0cd47d093e2fe76bbdf885a4984d6e